alacritty/src/logging.rs

```diff
@@ -127,8 +135,13 @@ impl log::Log for Logger {
     fn flush(&self) {}
 }
 
-fn create_log_message(record: &log::Record<'_>, target: &str) -> String {
-    let now = time::strftime("%F %T.%f", &time::now()).unwrap();
+fn create_log_message(record: &log::Record<'_>, target: &str, tz_offset: UtcOffset) -> String {
+    let now = OffsetDateTime::now_utc()
+        .to_offset(tz_offset)
+        .format(format_description!(
```

Comment:
If you move this `format_description!` out of the parameter, it'll clean stuff up:

```rust
    let time_format = format_description!(
        "[year]-[month]-[day] [hour repr:24]:[minute]:[second].[subsecond digits:9]"
    );
    let now = OffsetDateTime::now_utc().to_offset(tz_offset).format(time_format).unwrap();
```